cmd/openqa-mon/openqa-mon.go

```diff
@@ -157,8 +157,10 @@ func expandArguments(args []string) []string {
 				case 'c':
 					ret = append(ret, "--continuous")
 					// The next argument will be the number of seconds, add them here
-					ret = append(ret, args[i+1])
-					args[i+1] = ""
+					if i+1 < len(args) {
+						ret = append(ret, args[i+1])
+						args[i+1] = ""
+					}
```

Comment:
Thanks for the contribution! If we add already this check, I would also suggest to at least add a warning message, if it is not present. I'm not a big fan of silently swallowing user errors and prefer if programs yell at the user, if something is wrong :slightly_smiling_face: 
```suggestion
					} else {
					    fmt.Printf(os.Stderr, "missing time argument for --continuous");
					}
```

WDYT?

Reply:
actually with the out-of-bound check in place, the error message is handled by caller function, so it's not needed here. Could be better as an improvement to refactor the command line handling to return a value and an error :) 

with the fix in place:
```
/openqa-mon -c                                               
Missing continous period

```

Reply:
You're right, I completely forgot about that!